src/main/java/org/kiwiproject/dropwizard/error/config/CleanupConfig.java

```diff
@@ -24,12 +24,32 @@
     }
 
     /**
-     * The strategy to use for what to cleanup. Defaults to {@link CleanupStrategy#RESOLVED_ONLY}
+     * The strategy to use for what to cleanup. Defaults to {@link CleanupStrategy#ALL_ERRORS}
      */
-    private CleanupStrategy cleanupStrategy = CleanupStrategy.RESOLVED_ONLY;
+    private CleanupStrategy cleanupStrategy = CleanupStrategy.ALL_ERRORS;
 
     /**
-     * The duration of time that an error will live before being deleted.
+     * The duration of time that a resolved error will live before being deleted.
      */
-    private Duration applicationErrorExpiration = Duration.days(30);
+    private Duration resolvedErrorExpiration = Duration.days(14);
+
+    /**
+     * The duration of time that an unresolved error will live before being deleted.
+     */
+    private Duration unresolvedErrorException = Duration.days(60);
+
+    /**
+     * The name to give the scheduled job for cleaning up errors. Defaults to {@code Cleanup-Job-%d}
+     */
+    private String cleanupJobName = "Cleanup-Job-%d";
```

Comment:
For this, perhaps we actually do need more context since "Cleanup-Job" is pretty generic.

Maybe "Error-Cleanup-Job-%d" ?

"Application-Error-Cleanup-Job-%d" is also an option but is that a bit too long?

Could shorten to "App-Error-Cleanup-Job-%d".